### PR TITLE
Add GHG Emissions

### PR DESCRIPTION
**Pull request by haneslinger**

### CODE REVIEW

File: bsyncpy/bsync.py [hunk 1]
```diff
@@ -2365,7 +2395,16 @@ class EmissionsType(BSElement):
     """Category of greenhouse gas or other emission."""
 
     element_type = "xs:string"
-    element_enumerations = ["CO2e", "CO2", "CH4", "N2O", "NOx", "SO2", "Other"]
+    element_enumerations = [
+        "CO2e",
+        "MtCO2e",
```
Comment (JieXiong9119) [inline bsyncpy/bsync.py]: Need to wait confirmation on schema side if we want to add this enumeration or not. `MtCO2e` is more like a unit than a type of emissions.

File: bsyncpy/bsync.py [hunk 2]
```diff
@@ -2554,6 +2612,7 @@ class TimeSeriesReadingQuantity(BSElement):
         "Current Angle",
         "Demand",
         "Frequency",
+        "Greenhouse Gas Emissions",
```
Comment (haneslinger) [inline bsyncpy/bsync.py:2606]:
```suggestion
        "MtCO2e",
```

**Comment by haneslinger** [comment]
the formatting passes locally 😖 idk why they aren't passing

**Review by JieXiong9119 — APPROVED** [review]

**Comment by JieXiong9119** [comment]
> the formatting passes locally 😖 idk why they aren't passing

It looks like a poetry/pre-commit issue just appearing today. It's fixed on poetry side according to the issue [here](https://github.com/python-poetry/poetry/issues/5375), but I don't know how to fix this on our side.

**Comment by laurenliz22** [comment]
> > the formatting passes locally 😖 idk why they aren't passing
> 
> It looks like a poetry/pre-commit issue just appearing today. It's fixed on poetry side according to the issue [here](https://github.com/python-poetry/poetry/issues/5375), but I don't know how to fix this on our side.

I'm not sure how to fix this.  @kflemin would you have any ideas?

File: bsyncpy/bsync.py (hunk 1 above)
Comment (laurenliz22) [reply bsyncpy/bsync.py]: @JieXiong9119 we are not adding this enumeration in the schema.  Looks like it's been updated

**Review by laurenliz22 — APPROVED** [review]

**Comment by haneslinger** [comment]
Updating black like they did [here](https://github.com/TheAlgorithms/Python/pull/6075) fixed it

**Comment by JieXiong9119** [comment]
> Updating black like they did [here](https://github.com/TheAlgorithms/Python/pull/6075) fixed it

Nice! Looks like it's ready to go.

**Comment by kflemin** [comment]
Nice @haneslinger !!

**Comment by laurenliz22** [comment]
> > Updating black like they did [here](https://github.com/TheAlgorithms/Python/pull/6075) fixed it
> 
> Nice! Looks like it's ready to go.

Awesome, thanks @haneslinger!

**Review by nllong — APPROVED** [review]

**Pull request merged**